components/inputs/BeetsTokenInputWithSlider.tsx

```diff
@@ -168,7 +168,7 @@ export function BeetsTokenInputWithSlider({
                             cursor={hasBalance ? 'pointer' : 'default'}
                         >
                             You have {tokenFormatAmount(balance)}
-                            {hasBalance ? (
+                            {hasBalance && selectedTokenOption.address !== networkConfig.eth.address.toLowerCase() ? (
```

Comment:
Fine to remove max here, but ideally it would do what balancer does

Reply:
Doing it the balancer way.